### ISSUE TITLE
Adding/Consolidating Tutorial (Colab) for Metric Learning Losses 

### ISSUE THREAD

**Issue by shun-lin** [tensorflow/addons#1612]
There are many metric learning losses implemented here in TFA (focal loss, contrastive loss, lifted loss, npairs loss, triplet loss) and many potential ones that may be implemented in the future (Quad loss, multi-similarity loss, etc). It would be great to have a tutorial on this :)

**Relevant information**
- Are you willing to contribute it (yes/no): Yes
- Are you willing to maintain it going forward? (yes/no): Yes (well it's just a tutorial)
- Is there a relevant academic paper? (if so, where): N/A
- Is there already an implementation in another framework? (if so, where): N/A
- Was it part of tf.contrib? (if so, where): N/A

**Comment by ashutosh1919** [comment]
<img width="171" alt="contrastive_img" src="https://user-images.githubusercontent.com/20843596/80303529-6eca8400-87ce-11ea-89cf-36e07ceff412.PNG">

<img width="221" alt="contrastive_formula" src="https://user-images.githubusercontent.com/20843596/80303725-8f470e00-87cf-11ea-9e61-0eb226a6568e.PNG">

<img width="245" alt="focal_img" src="https://user-images.githubusercontent.com/20843596/80304296-873c9d80-87d2-11ea-8f0f-830b85bff142.PNG">

<img width="129" alt="focal_formula" src="https://user-images.githubusercontent.com/20843596/80304383-0336e580-87d3-11ea-9cd8-e51e7377f999.PNG">

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
